### PR TITLE
LPS-94472 layout-test: Create LayoutMultiLanguageSearchTest

### PR DESCRIPTION
**Pull request by brandizzi**

### CODE REVIEW

**Comment by foshiro-zz** [comment]
ci:test:sf

**Comment by foshiro-zz** [comment]
ci:test:search

File: modules/apps/layout/layout-test/src/testIntegration/java/com/liferay/layout/search/test/LayoutFixture.java [hunk 1]
```diff
@@ -45,6 +47,35 @@ public Layout createLayout() throws PortalException {
...
+						FriendlyURLNormalizerUtil.normalize(
+							RandomTestUtil.randomString()));
+			}
+		};
+
+		Layout layout = LayoutLocalServiceUtil.addLayout(
+			TestPropsValues.getUserId(), _group.getGroupId(), false,
+			LayoutConstants.DEFAULT_PARENT_LAYOUT_ID, nameMap.getValues(),
+			titleMap.getValues(), null, null, null,
+			LayoutConstants.LAYOUT_TYPE_CONTENT, null, false,
```
Comment (foshiro-zz) [inline modules/apps/layout/layout-test/src/testIntegration/java/com/liferay/layout/search/test/LayoutFixture.java]: This constant was renamed. Please change to TYPE_CONTENT

File: modules/apps/layout/layout-test/src/testIntegration/java/com/liferay/layout/search/test/LayoutMultiLanguageSearchTest.java [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+ */
+@RunWith(Arquillian.class)
+public class LayoutMultiLanguageSearchTest {
+
+	@ClassRule
+	@Rule
+	public static final AggregateTestRule aggregateTestRule =
+		new AggregateTestRule(
+			new LiferayIntegrationTestRule(),
+			PermissionCheckerTestRule.INSTANCE,
```
Comment (foshiro-zz) [inline modules/apps/layout/layout-test/src/testIntegration/java/com/liferay/layout/search/test/LayoutMultiLanguageSearchTest.java]: Please change to PermissionCheckerMethodTestRule

**Review by foshiro-zz — CHANGES_REQUESTED** [review]
@vagner-castro , code looks good. Just have to change some constants and methods that were renamed. Please check my comments

**Comment by vagnerbc** [comment]
Changes updated

**Comment by foshiro-zz** [comment]
LGTM

**Comment by foshiro-zz** [comment]
Submitted to https://github.com/brandizzi/liferay-portal/pull/770

**Pull request closed without merging**